src/richie-front/js/components/CourseGlimpse/CourseGlimpse.tsx

```diff
@@ -35,22 +35,25 @@ export const CourseGlimpse = injectIntl(
     const { course, intl, organizationMain } = props;
 
     return (
-      <div className="course-glimpse-container">
-        <div className="course-glimpse">
+      <a className="course-glimpses__item course-glimpses__item--link" href="#">
+        <div className="course-glimpses__item__media">
           <img
-            className="course-glimpse__image"
             src={'https://www.fun-mooc.fr' + course.thumbnails.small}
             alt={intl.formatMessage(messages.altText, {
               courseTitle: course.title,
             })}
           />
-          <div className="course-glimpse__body">
-            <div className="course-glimpse__body__title">{course.title}</div>
-            <div className="course-glimpse__body__org">
+        </div>
+        <div className="course-glimpses__item__content">
+          <div className="course-glimpses__item__wrapper">
+            <p className="course-glimpses__item__content__title">{course.title}</p>
+            <p>
               {(organizationMain && organizationMain.name) || ''}
-            </div>
+            </p>
           </div>
-          <div className="course-glimpse__date">
+        </div>
+        <div className="course-glimpses__item__foot">
```

Comment:
`footer` rather than `foot` ?

Reply:
Commonly we keep header and footer for the page ones, for sub element and component we should stick to head/foot

Reply:
@sveetch head/foot is not the proper term for this I agree that we should stick to header/footer.